drivers/hmis/app/graphql/mutations/update_project.rb

```diff
@@ -25,8 +25,9 @@ def create_warnings(project, input)
 
       funder_count = project.funders.where(end_date: nil).count
       inventory_count = project.inventories.where(inventory_end_date: nil).count
+      open_enrollments = Hmis::Hud::Enrollment.open_on_date.in_project(project.id)
       warnings = []
-      # TODO add warning if there are open enrollments in project
+      warnings << InputConfirmationWarning.new("#{open_enrollments.count} open #{'enrollment'.pluralize(open_enrollments.count)} will be closed.", attribute: 'id') if open_enrollments.present?
```

Comment:
We don't actually close the enrollments, we just want to let the user know they exist
```suggestion
      warnings << InputConfirmationWarning.new("Project has #{open_enrollments.count} open #{'enrollment'.pluralize(open_enrollments.count)}.", attribute: 'id') if open_enrollments.present?
```

Reply:
Changed copy